### PR TITLE
Aide cron

### PR DESCRIPTION
**Pull request by hooksie1**
Adds cron job to the aide_build_database.yml playbook

### CODE REVIEW

**Comment by redhatrises** [comment]
@hooksie1 please but this in a separate file called "aide_periodic_cron_checking.yml" as this matches the corresponding rule and oval check.

**Comment by yuumasato** [comment]
@openscap-ci test this please

File: shared/templates/static/ansible/aide_periodic_cron_checking.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: Schedule AIDE cron job"
```
[the remark this answers is not in the document]
Comment (hooksie1) [reply shared/templates/static/ansible/aide_periodic_cron_checking.yml]: Ok I think I got it. Sorry about the other file, I thought I reverted it previously.

**Comment by yuumasato** [comment]
@hooksie1 Hello, resulting changes are ok, but the commits to get there are messy.
Can you remove the commits that touch `aide_build_database.yml`? There is a also merge commit that should not be there.
Would you like some guidance? Although I'm not sure I'm able to give you exact commands.

**Comment by hooksie1** [comment]
Sorry I got super busy this month. I'm just going to close this one and I'll open another. I can't figure out what I screwed up with ha.

**Pull request closed without merging**